src/content/basics/aws-architecture/index.md

```diff
@@ -5,7 +5,24 @@ date: 2020-06-15
 weight: 50
 type: page
 categories: ["basics"]
-last-review-date: 2020-05-18
+last-review-date: 2020-10-28
+user_questions:
+  - What areas are covered in the Giant Swarm AWS platform?
+  - Do I need a control plane per AWS region?
+  - What does Giant Swarm use operators for?
+  - How does Ginat Swarm Manage Custom Resources (CRs)?
+  - How can users of Giant Swarm manage their clsuters?
+  - How can users of Giant Swarm manage their applications?
+  - How does Giant Swarm operate in the AWS landscape?
+  - Why does Giant Swarm need access to my AWS account?
+  - What isolation layers are available when using Giant Swarm on AWS?
+  - What are best practices for workload segregation on AWS?
+  - How would Node Pools look on tenant clusters in AWS?
+  - What are best practices around the size of worker nodes?
```

Comment:
I think for this question we have a more specific place: https://docs.giantswarm.io/basics/cluster-size-autoscaling/

Reply:
There are two paragraphs that refer to this very explicitly in this document. In addition, the wording is different between the documents. Is it a problem that two documents would come up in the search?

Reply:
(In case you wonder: this comment was on line 21)

I would opt for placing the question on the page that provides the best answer to it.